docs/lotion/overview.md

```diff
@@ -1,5 +1,54 @@
-# Lotion JS Overview
+# Overview
 
-Lotion is a new way to create blockchain apps in JavaScript, which aims to make writing new blockchains fast and fun. It builds on top of Tendermint using the ABCI protocol. Lotion lets you write secure, scalable applications that can easily interoperate with other blockchains on the Cosmos Network using IBC.
+Lotion is an alternative to the Cosmos SDK and allows you to create blockchain apps in JavaScript. It aims to make writing new blockchain apps fast and fun, and builds on top of Tendermint using the ABCI protocol. Lotion lets you write secure, scalable applications that can easily interoperate with other blockchains on the Cosmos Network using IBC.
```

Comment:
I'd rephrase:
"It aims to make writing new blockchain apps easy and fast by using the ABCI protocol to build on top of Tendermint."